google/gax/api_callable.py

```diff
@@ -88,7 +88,7 @@ def _retryable(a_func, retry):
     total_timeout = (retry.backoff_settings.total_timeout_millis /
                      _MILLIS_PER_SECOND)
 
-    def inner(*args, **kwargs):
```

Comment:
This is a good cleanup -- previously, both CallOptions and the inner call took `kwargs`, which was redundant. This is better.